### PR TITLE
Task/jenkins 43740 events test coverage improvement

### PR DESCRIPTION
**Pull request by vivek**
# Description

See [JENKINS-43740](https://issues.jenkins-ci.org/browse/JENKINS-43740).

Most of SSE events related code and functionality is covered now. class coverage at 100% and line coverage at 91%.

image

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [x] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [ ] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

**Comment by i386** [comment]
LGTM

**Comment by michaelneale** [comment]
if the tests can be cleaned up - by all means, good to get this in.

File: blueocean-events/src/test/java/io/jenkins/blueocean/events/sse/SSEConnection.java [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+import java.util.concurrent.Executor;
+
+/**
+ * Represents an SSE connection to a server.
+ *
+ * @author Kohsuke Kawaguchi
+ * @author Vivek Pandey
+ * @see <a href="https://www.w3.org/TR/eventsource/">spec</a>
+ */
+public class SSEConnection implements Closeable {
```
Comment (michaelneale) [inline blueocean-events/src/test/java/io/jenkins/blueocean/events/sse/SSEConnection.java]: is this a refactoring from somewhere else?

Comment (vivek) [inline blueocean-events/src/test/java/io/jenkins/blueocean/events/sse/SSEConnection.java]: Yes, we did as part of blue steel poc. I changed it to use as test quality utility.

**Pull request merged**